torch/linalg/__init__.py

```diff
@@ -140,6 +140,111 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+svd = _add_docstr(_linalg.linalg_svd, r"""
+linalg.svd(input, full_matrices=True, compute_uv=True, out=None) -> (Tensor, Tensor, Tensor)
+
+This function returns a namedtuple ``(U, S, Vh)`` which is the singular value
```

Comment:
What about something like:

"Computes the singular value decomposition of either a matrix or batch of matrices :attr:\`input\`." The singular value decomposition is represented as a namedtuple \`\`(U, S, Vh)\`\`, such that :math:\`input = U \times diag(S) \times Vh\`."

There are still a couple problems with that proposal, however.

First, how is the singular value decomposition for a batch returned? Is it the same triple but U, S, and Vh have the batch dimensions, too? The above should include that info.

Second, full_matrices and compute_uv need to be described. That can be done in a paragraph immediately after this opening.

Reply:
note that currently it is phrased in the very same way as `torch.svd`. I think we should try to minimize the differences between the two, so that it is immediately clear to the user what is the same and what is different. If we change this, I vote to change `torch.svd` accordingly.

Moreover, if we really want to change the wording, I propose that we just copy&adapt the [numpy docs](https://numpy.org/doc/stable/reference/generated/numpy.linalg.svd.html). Personally, when I started this PR I had to spend some time to read carefully the docs of both to try to understand whether they were equivalent or there were subtle differences, so I think all our users will be grateful if we try to be as close as possible.

> Second, full_matrices and compute_uv need to be described. That can be done in a paragraph immediately after this opening.

they are already described in a paragraph after the opening :)